docs/configuration.rst

```diff
@@ -97,6 +97,12 @@ A few relevant settings for your `Pyramid .ini file <http://docs.pylonsproject.o
         pyramid_swagger.dereference_served_schema = false
 
 
+.. note::
+
+    ``pyramid_swawgger`` uses a ``bravado_core.spec.Spec`` instance for handling swagger related details.
+    It is possible to tune ``bravado-core`` configs adding ``bravado_core.`` prefix to the ``bravado-core`` config keys.
```

Comment:
"You can set \`bravado-core config values \<http://bravado-core.readthedocs.io/en/stable/config.html>`_ by adding a \`\`bravado-core.\`\` prefix to them."